crates/ruff/src/rules/pylint/rules/redefined_loop_name.rs

```diff
@@ -7,53 +7,10 @@ use ruff_diagnostics::{Diagnostic, Violation};
 use ruff_macros::{derive_message_formats, violation};
 use ruff_python_ast::comparable::ComparableExpr;
 use ruff_python_ast::statement_visitor::{walk_stmt, StatementVisitor};
-use ruff_python_ast::types::Node;
 use ruff_python_semantic::SemanticModel;
 
 use crate::checkers::ast::Checker;
 
-#[derive(Debug, PartialEq, Eq, Clone, Copy)]
-pub(crate) enum OuterBindingKind {
```

Comment:
Made these private, moved them below the violation (so the violation appears first in the module).